recipes/types-sourceforge/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "types-scriptforge" %}
+{% set version = "1.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/types-scriptforge-{{ version }}.tar.gz
+  sha256: 4203e284e99a5daee31eca99b1501bb958b5a5c0fb44a23952f5304fd48b1db2
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - types-unopy >=0.3.10
+    - typing-extensions >=3.7.4.3
```

Comment:
```suggestion
```
Upstream says that typing-extensions is only required for python<3.7, but you have restricted your package to python>=3.7, so typing-extensions is not required.

https://github.com/Amourspirit/python-types-scriptforge/blob/e04899cb5a806ae7c9b8e9743fbad533bbea20aa/setup.py#L43